virttest/test_setup.py

```diff
@@ -2523,6 +2524,7 @@ def enable(self):
                                                 "libvirtd.log")
         # param used during libvirtd cleanup
         self.test.params["libvirtd_debug_file"] = self.log_file
+        Path(self.log_file).touch(mode=0o644, exist_ok=True)
```

Comment:
Could we reuse current module method os.chmod(), rather than Path to achieve since os module is already imported?

Reply:
To use `os.chmod()`, we should ensure the file exists. But before the daemon is restarted and collects log contents, we cannot assume it exists. A possible location is when restoring the daemon, https://github.com/avocado-framework/avocado-vt/blob/9dfc42808ac5ed2deab20f2b91c3bad5e126b75e/virttest/test_setup.py#L2545-L2549

But it seems no reason to custom log permission when disabling daemon, and LibvirtdDebugLog.enable in `preprocess` and `LibvirtdDebugLog.disable` in `postprocess`.